invenio_app_ils/ill/schemas/ill-borrowing-requests/borrowing-request-v1.0.0.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Schema for the ILL borrowing request",
+  "required": [
+    "$schema",
+    "pid",
+    "status",
+    "library_pid"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Interlibrary borrowing request schema",
+      "type": "string"
+    },
+    "pid": {
+      "title": "The persistent identifier of ILL borrowing request",
+      "type": "string"
+    },
+    "status": {
+      "title": "Status of the ILL borrowing request",
+      "type": "string"
+    },
+    "cancel_reason": {
```

Comment:
Is this the same as the `reject_reason` in document-requests?

Reply:
Yes!